doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -713,8 +713,65 @@ so notification receiver can know the format used without needing processing not
 
 ### Notification templates
 
-Based on the "template" field defined as part of the subscription and in a macro replacement language
-that could be used in "payload", "url", "headers", "method" and "qs". Details TBD.
```

Comment:
I would say here:

NGSIv2 clients can customize HTTP notification messages using a template language. The notification.httpCustom field allows to specify the following fields subject to be templatized: .... 

In addition the HTTP method to be used for delivering the notification can be customized as well, but it cannot be a template.